scss/_modal.scss

```diff
@@ -140,3 +140,7 @@
 @include media-breakpoint-up(lg) {
   .modal-lg { max-width: $modal-lg; }
```

Comment:
Maybe we can add a line below to here for smooth transition between md and lg break points.
  .modal-xl { max-width: $modal-lg; }